codespell_lib/data/dictionary.txt

```diff
@@ -43433,9 +43520,18 @@ relocateable->relocatable
 reloccate->relocate
 reloccated->relocated
 reloccates->relocates
+relpace->replace
+relpaced->replaced
 relpacement->replacement
+relpacements->replacements
+relpaces->replaces
+relpacing->replacing
 relpy->reply
 reltive->relative
+relvance->relevance
+relvancy->relevancy
+relvant->relevant
```

Comment:
Perhaps `reliant` too, although **I** and **V** are not neighbours on my keyboard?

Reply:
I couldn't find this typo.